src/box/lua/load_cfg.lua

```diff
@@ -109,6 +109,7 @@ local default_cfg = {
     feedback_interval     = 3600,
     net_msg_max           = 768,
     sql_cache_size        = 5 * 1024 * 1024,
+    txn_timeout           = 2,
```

Comment:
@unera said that the default value should be equal to 'infinity' AFAIR. Also, he said that we should give users the ability to overwrite the default timeout. Need to agree on the API. I see the following options:

1. 
   ```lua
   box.begin(123)
   ```
2.
   ```lua
   box.begin{timeout = 123}
   ```
3.
   ```lua
   box.begin()
   box.txn_timeout(123)
   ```

I like option no. 2.

Reply:
Fixed